src/expr/proof_rule.h

```diff
@@ -727,6 +727,47 @@ enum class PfRule : uint32_t
   // ---------------------
   // Conclusion: (= t bitblast(t))
   BV_BITBLAST,
```

Comment:
We can delete this now?

Reply:
It looks like it. @mpreiner and I weren't sure initially if we wouldn't need it for something.

Reply:
Shouldn't be needed anymore since we now have the fine-grained rules.

Reply:
We discussed this offline, will keep it for coarse proofs.